force-app/main/default/classes/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,26 @@
+public with sharing class ContactTriggerHandler {
+
+    // Future method to make a callout to the getDummyJSONUserFromId API
+    @future(callout=true)
```

Comment:
I like that you choose to make the TriggerHandler method @future and left your DummyJSONCallout class methods regular, makes it a little more flexible if you want to call your callout methods synchronously from other way.